<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line Fitting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="show_win">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>447</y>
      <width>851</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="dashboard">
     <item>
      <layout class="QHBoxLayout" name="fit_equation">
       <item>
        <widget class="QLabel" name="fit_eq_label">
         <property name="text">
          <string> Fit equation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="fit_eq"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="btns_windows">
       <item>
        <layout class="QVBoxLayout" name="btns">
         <item>
          <widget class="QPushButton" name="btn_load">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_clean">
           <property name="text">
            <string>Clean</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="show_eq"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
